diff --git a/lib/index.js b/lib/index.js
index f3715e08e37c90ac80cd8d349112d6fa57fa8db9..07ba528ffcd23fc46fdf8eeb00605df6813b424a 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -181,7 +181,7 @@ const recursivelyCollectAllDependencies = (options) => {
     /*istanbul ignore else*/
     if (clarification?.licenseFile) {
         licenseFilesInCurrentModuleDirectory = [clarification.licenseFile];
-    } else if (fs.existsSync(modulePath)) {
+    } else if (typeof modulePath === 'string' && fs.existsSync(modulePath)) {
         const filesInModuleDirectory = fs.readdirSync(modulePath);
         licenseFilesInCurrentModuleDirectory = licenseFiles(filesInModuleDirectory);
 
